src/components/cc-orga-member-list/cc-orga-member-list.js

```diff
@@ -0,0 +1,632 @@
+      }
+      else {
+        suggestedElement.focusDeleteBtn();
+      }
+    });
+  }
+
+  /**
+   * Check if the member is the last admin of the organisation.
+   *
+   * @param {OrgaMemberState} member - the member to check
+   * @return {boolean} - true if the current user is an admin and there is only one admin left in the organisation
+   */
+  _checkIsLastAdmin (member) {
+    const isCurrentUserAdmin = member.role === 'ADMIN';
+    const adminList = this._getAdminList();
+    const orgContainsOnlyOneAdmin = adminList.length === 1;
+
+    return orgContainsOnlyOneAdmin && isCurrentUserAdmin;
+  }
+
+  /**
+   * @return {OrgaMemberCardState[]} AdminMembers - the list of admins in the organisation. Used to ensure we don't allow deletion of the last admin.
+   */
+  _getAdminList () {
+    return this.members.value.filter((member) => member.role === 'ADMIN');
+  }
+
+  _getRoleOptions () {
+    return [
+      { value: 'ADMIN', label: i18n('cc-orga-member-list.invite.role.admin') },
+      { value: 'DEVELOPER', label: i18n('cc-orga-member-list.invite.role.developer') },
+      { value: 'ACCOUNTING', label: i18n('cc-orga-member-list.invite.role.accounting') },
+      { value: 'MANAGER', label: i18n('cc-orga-member-list.invite.role.manager') },
+    ];
+  }
+
+  /**
+   * Everytime a new list is rendered, this method is called to:
+   * - sort the list (current user first, then by member email in alphabetical order),
+   * - filter the list based on what the user has entered inside the identity and mfa filters.
+   *
+   * @param {OrgaMemberCardState[]} members
+   * @param {string} identityFilter
+   * @param {boolean} mfaDisabledOnlyFilter
+   * @return {OrgaMemberCardState[]}
+   * @private
+   */
+  _getSortedFilteredMemberList (members, identityFilter, mfaDisabledOnlyFilter) {
+    const cleanedIdentityFilter = identityFilter?.trim().toLowerCase();
+
+    const filteredMemberList = members.filter((member) => {
+      const matchIdentity = cleanedIdentityFilter === ''
+        || member.name?.toLowerCase().includes(cleanedIdentityFilter)
+        || member.email.toLowerCase().includes(cleanedIdentityFilter);
+
+      const matchMfaDisabled = !mfaDisabledOnlyFilter || !member.isMfaEnabled;
+
+      return matchIdentity && matchMfaDisabled;
+    });
+
+    const sortedFilteredMemberList = filteredMemberList.sort((a, b) => {
+      // currentUser first, then alphabetical order - case-insensitive
+      if (a.isCurrentUser) {
+        return -1;
+      }
+      if (b.isCurrentUser) {
+        return 1;
+      }
+      return a.email.localeCompare(b.email, { sensitivity: 'base' });
+    });
+
+    return sortedFilteredMemberList;
+  }
+
+  /**
+   * Set or remove an error of the member card corresponding to the provided member id.
+   *
+   * @param {string} memberId - the id corresponding to the member card
+   * @param {boolean} errorType - whether the card should display an error message or not
```

Comment:
Also, it seems like you're only using this `_setMemberCardError` twice and the second param is always `true`.

Reply:
Yes, since we no longer show errors within the card when managers try to temper with an admin, this method doesn't make much sense.

I removed it and I set the errors directly from `_onUpdateFromCard` and `_onLeave...` methods.